src/net_processing.cpp

```diff
@@ -2476,19 +2476,12 @@ void PeerLogicValidation::ProcessMessage(CNode& pfrom, const std::string& msg_ty
             // We skip these operations for BLOCK_RELAY peers to avoid
             // potentially leaking information about our BLOCK_RELAY
             // connections via the addrman or address relay.
-            if (fListen && !::ChainstateActive().IsInitialBlockDownload())
+            if (fListen && !m_chainman.ActiveChainstate().IsInitialBlockDownload())
```

Comment:
Please don't change `::ChainstateActive` to `m_chainman.ActiveChainstate` in this PR. It adds noise for reviewers.